.github/workflows/build_and_test.yml

```diff
@@ -49,5 +49,8 @@ jobs:
     - name: Build
       run: go build -v ./...
 
-    - name: Test
-      run: go test -v ./...
+    - name: Run Unit Tests
+      run: go test ./{/otel,/spanner,/tableConfig,/tableConfig,/translator,/utilities} 
+
+    - name: Run Third Party Unit Tests
```

Comment:
This is still failing, I would suggest to remove this from this PR and address it in the next PR.

Reply:
Resolving as all tests are passing now.